src/main/python/interfaces/helper.py

```diff
@@ -74,3 +74,14 @@ def menu_icon(width):
     painter.end()
 
     return QIcon(pix)
+
+def get_time():
+
+    morning = [f"{i} AM" for i in range(1, 13)]
+    afternoon = [f"{i} PM" for i in range(1, 13)]
```

Comment:
this is almost what I was thinking of but not quite.  Because these lists will never change, we don't have to allocate them at the time that the function is called.  Right now, every time get_time is called, we're allocating `morning` and `afternoon`, then returning a reference to the concatenated list.  However, we could easily make these defined at the time of import, by creating a variable at the top of `helper.py`, something like:
```
times = [f"{i} AM" for i in range(1, 13)] + [f"{i} PM" for i in range(1, 13)]
```

and then have `get_time` just return that list.  That way we'd only allocate it once and then return a reference to it every time following that.  I'm pretty sure this is how python would do it, but I'm not 100% sure since python does some clever optimizations.  What do you think @fennnn @lixphilosophy @samgomena

Reply:
actually I just confirmed it, we're allocating new objects every time that funciton is called.  Better to allocate once then return rather than allocate every time:

```
>>> a = [f"{i}" for i in range(1, 3)] + [f"{j}" for j in range(5,77)]
>>> def b():
...     c = [1, 2, 3, 4]
...     d = [6,2,3,4]
...     f = c + d
...     return id(f)
>>> b()
53507896
>>> b()
53506576
>>> b()
21988656
>>> b()
53507896
>>> b()
53506576
>>> def r_a():
...     return id(a)
>>> r_a()
53508016
>>> r_a()
53508016
>>> r_a()
53508016
>>> r_a()
53508016
>>> r_a()
53508016
>>> r_a()
53508016
```

Reply:
Agreed. For that matter, it might be simpler to just define `HOURS` and `DAYS` as constants that can then be imported when needed.

Reply:
Fixed